.github/pull_request_template.md

```
@@ -0,0 +1,29 @@
+## 📌 Summary
+
+>  #️⃣ 연관된 이슈
```

markdownlint-cli2 (0.18.1)

Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)